detox/test/index.ios.js

```diff
@@ -10,46 +10,97 @@ import {
   StyleSheet,
   Text,
   View,
+  ScrollView,
   TouchableOpacity
 } from 'react-native';
+import _ from 'lodash';
+
+const STRESSFUL_STRING_LENGTH = 7800
```

Comment:
our conventions is to have ; in the end of all lines, please fix in all the code